cypress.json

```diff
@@ -1 +1,4 @@
-{}
+{
+    "chromeWebSecurity": false
```

Comment:
@mikeboyle 🤷 . I don't know why the security errors are happening, since this isn't attempting anything with iframes... but you can see from the CI run that the non-bonus spec passes now.